auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -25,6 +25,8 @@
         </div>
       </div>
 
+      <DashboardHelp/>
```

Comment:
Not require but better to have a space before `/>`. (If there were properties in there, you'd want the space for sure.)